ArtOfIllusion/src/artofillusion/view/ClickedPointFinder.java

```diff
@@ -0,0 +1,186 @@
+/* Copyright (C) 2016 by Petri Ihalainen
+   Changes Copyright 2016 by Petri Ihalainen
+
+   This program is free software; you can redistribute it and/or modify it under the
+   terms of the GNU General Public License as published by the Free Software
+   Foundation; either version 2 of the License, or (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful, but WITHOUT ANY
+   WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+   PARTICULAR PURPOSE.  See the GNU General Public License for more details. */
+
+package artofillusion.view;
+
+import artofillusion.math.*;
+import artofillusion.ui.*;
+import artofillusion.object.*;
+import artofillusion.*;
+import java.awt.*;
+
+public class ClickedPointFinder
+{
+	private TriangleMath T = new TriangleMath();
```

Comment:
You should not have to do this. You're creating a dummy object to get access to methods that do not operate on an internal data structure. This follows along with the comments regarding making TriangleMath methods static.